### ISSUE TITLE
Docs are not building.

### ISSUE THREAD

**Issue by jyutzio** [jyutzio/safi#1]
```bash
> npx styleguidist build

Loading webpack config from:
/home/jyutzio/Documents/Programming/Projects/safi/webpack.config.js

Building style guide...
CopyPlugin Invalid Options

options should NOT have fewer than 1 items

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! safi@0.0.1 build-docs: `npx styleguidist build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the safi@0.0.1 build-docs script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jyutzio/.npm/_logs/2019-12-10T23_35_13_006Z-debug.log
The terminal process terminated with exit code: 1

Terminal will be reused by tasks, press any key to close it.
```

**Comment by jyutzio** [comment]
Fixed in #02d138617e8199a7c4a3aa41d3b2330e0e061747

**Comment by jyutzio** [comment]
a9e3555e2db750283766e951e58c33560b7822df
